templates/zerver/find_account.html

```diff
@@ -56,12 +36,11 @@ <h1 class="get-started">{{ _("Find your Zulip accounts") }}</h1>
                 </form>
                 <div id="errors"></div>
                 {% if form.emails.errors %}
-                    {% for error in form.emails.errors %}
-                    <div class="alert alert-error">{{ error }}</div>
-                    {% endfor %}
+                {% for error in form.emails.errors %}
+                <div class="alert alert-error">{{ error }}</div>
+                {% endfor %}
```

Comment:
The indentation was fine, i guess ?

Reply:
The lint check didn't throw any errors so ignored it